devdocs/settingUp.md

```diff
@@ -166,19 +171,28 @@ Refer to [this document](staticAnalysis.md) for details on the tools used, how t
 
 To change the browser that is used in the UI tests, go to the `test.properties` 
 file and change the `test.selenium.browser` value to the browser you want to test. 
-Currently only `firefox` is accepted.
+Possible values are `firefox` and `chrome`.
 In addition, you need to configure the browser you have selected so that 
 it works with the test suite. 
 
 ####Firefox
 
-* If you are planning to test changes to JavaScript code, disable 
-  javascript caching for Firefox - Enter `about:config` into the 
-  Firefox address bar and set: `network.http.use-cache = false`
+* If you are planning to test changes to JavaScript code, disable javascript caching for Firefox:
+    * Enter `about:config` into the Firefox address bar and set `network.http.use-cache = false`.
 * If you have installed a separate Firefox version, you can choose which 
   Firefox binary to use. You can specify the custom path in `test.firefox.path` 
   value inside the `test.properties` file.
 
+####Chrome
+* If you are planning to test changes to JavaScript code, disable javascript caching for Chrome:
+    * Press Ctrl+Shift+J to bring up the Web Console.
+    * Click on the settings button at the bottom-right corner.
+    * Under the General tab, check 'Disable Cache'.
+* The chromedriver process started by the test suite will not automatically get killed after the tests have finished executing.
+  You will need to manually kill these processes after the tests are done.
+  On Windows, you can do this using the Task Manager or `taskkill /f /im chromedriver.exe` command.
+  On Mac, use the Activity Monitor or `sudo killall chromedriver` command in the similar manner.
```

Comment:
Somehow I didn't read this part when testing it out, so I had 7 processes running in the background for the past few hours :scream: